capstone/src/components/User/Adventurer/ExperienceInfo/ExperienceInfo.jsx

```diff
@@ -25,6 +51,42 @@ export default function ExperienceInfo({ restaurants }) {
     ? getPhotoReference(currentRestaurant)
     : null;
 
+  // get active feedback information
+  useEffect(() => {
+    if (currentRestaurant) {
+      getActiveFeedback(currentRestaurant.place_id).then((res) =>
+        setActiveFeedback(res.data),
+      );
+      getIsRated(username, currentRestaurant.place_id).then((res) =>
+        setIsRated(res.data),
+      );
+    }
```

Comment:
(low-priority) It might be easier if we had a single endpoint that gave us all info about a restaurant.